bazel/hermetic_cc_toolchain.patch

```diff
@@ -0,0 +1,210 @@
+# Extend hermetic_cc_toolchain with a wasm target that does not use wasi, and
+# add a path to put additional restrictions on the generated toolchains.
```

Comment:
can you create a ticket or PR upstream? we're definitely not the only ones building for wasm without wasi